circle.yml

```diff
@@ -49,6 +49,9 @@ commands:
           working_directory: /tmp
           command: |
             if [ ! -f testfloat_gen ]; then
+              # GCC compiler is expected, create link from clang if gcc not available.
+              type gcc || (cd $(dirname $(which clang)) && ln -s clang gcc)
```

Comment:
So what problem is this solving if the CI was not breaking? Some future PR?

Reply:
This problem is not visible usually, because we take the binary from cache. But cache is disabled for external PRs (like this one) and then jobs having clang compiler only fails. The makefile from TestFloat is using `gcc` for compiler instead of `cc`.